code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/filter/ServiceCreateValidationFilter.java

```diff
@@ -49,6 +60,30 @@ public Object create(String type, ApiRequest request, ResourceManager next) {
                     "name");
         }
 
+        // 2. if a part of sidekick, scale should match other sidekick participants scale
+        Map<String, Object> launchConfig = DataAccessor
```

Comment:
as per our hangouts discussion, at the moment service gets created and sidekick is specified, its scale will get set to the existing sidekick's services scale